packages/cli/src/commands/setup/deploy/templates/render.js

```diff
@@ -5,6 +5,12 @@ import { getPaths } from '../../../../lib'
 export const PROJECT_NAME = path.basename(getPaths().base)
 
 export const RENDER_YAML = (database) => {
+  // sqlite requires api to be plan of starter
+  // if database starts with "file:./" lets 
```

Comment:
@jacebenson There are several `lint` warnings (note: not errors) in the code. Could you please resolve?

Reply:
Yes, working on that.  Also I added a console.log when database is sqlite to say
> Sqlite requires a services plan of starter which incurs a cost on Render.